frontend/src/views/Transactions/Transactions.jsx

```diff
@@ -225,6 +225,8 @@ export const Transactions = () => {
           handleGridKey={handleGridKey}
           handleRowClicked={handleRowClicked}
           enableCopyButton={false}
+          enableExportButton={true}
```

Comment:
Let's just remove this feature from transactions but can think about other tables we could use it.

Reply:
Sure, thank you for reviewing my PR.